deepchem/models/torch_models/cnn.py

```diff
@@ -0,0 +1,441 @@
+
+    if mode not in ['classification', 'regression']:
+      raise ValueError("mode must be either 'classification' or 'regression'")
+
+    self.n_tasks = n_tasks
+    self.n_features = n_features
+    self.dims = dims
+    self.mode = mode
+    self.n_classes = n_classes
+    self.uncertainty = uncertainty
+    self.mode = mode
+    self.layer_filters = layer_filters
+    self.residual = residual
+
+    n_layers = len(layer_filters)
+
+    # PyTorch layers require input and output channels as parameter
+    # if only one layer to make the model creating loop below work, multiply layer_filters wutg 2
+    if len(layer_filters) == 1:
+      layer_filters = layer_filters * 2
+
+    if not isinstance(kernel_size, SequenceCollection):
+      kernel_size = [kernel_size] * n_layers
+    if not isinstance(strides, SequenceCollection):
+      strides = [strides] * n_layers
+    if not isinstance(dropouts, SequenceCollection):
+      dropouts = [dropouts] * n_layers
+    if not isinstance(activation_fns, SequenceCollection):
+      activation_fns = [activation_fns] * n_layers
+    if not isinstance(weight_init_stddevs, SequenceCollection):
+      weight_init_stddevs = [weight_init_stddevs] * n_layers
+    if not isinstance(bias_init_consts, SequenceCollection):
+      bias_init_consts = [bias_init_consts] * n_layers
+
+    self.activation_fns = [get_activation(f) for f in activation_fns]
+
+    if uncertainty:
+
+      if mode != 'regression':
+        raise ValueError("Uncertainty is only supported in regression mode")
+
+      if any(d == 0.0 for d in dropouts):
+        raise ValueError(
+            'Dropout must be included in every layer to predict uncertainty')
+
+    # Python tuples use 0 based indexing, dims defines number of dimension for convolutional operation
+    ConvLayer = (nn.Conv1d, nn.Conv2d, nn.Conv3d)[self.dims - 1]
+
+    if pool_type == 'average':
+      PoolLayer = (F.avg_pool1d, F.avg_pool2d, F.avg_pool3d)[self.dims - 1]
+    elif pool_type == 'max':
+      PoolLayer = (F.max_pool1d, F.max_pool2d, F.max_pool3d)[self.dims - 1]
+    else:
+      raise ValueError("pool_type must be either 'average' or 'max'")
+
+    self.PoolLayer = PoolLayer
+    self.layers = nn.ModuleList()
+
+    in_shape = n_features
+
+    for out_shape, size, stride, weight_stddev, bias_const, dropout in zip(
+        layer_filters, kernel_size, strides, weight_init_stddevs,
+        bias_init_consts, dropouts):
+
+      block = nn.Sequential()
+
+      layer = ConvLayer(in_channels=in_shape,
+                        out_channels=out_shape,
+                        kernel_size=size,
+                        stride=stride,
+                        padding=padding,
+                        dilation=1,
+                        groups=1,
+                        bias=True)
+
+      nn.init.normal_(layer.weight, 0, weight_stddev)
+
+      layer_bias: torch.Tensor = layer.bias
+      nn.init.constant_(layer_bias, bias_const)
```

Comment:
Why not use layer.bias in nn.init.constant_ rather than define a new variable layer_bias?

Reply:
I did, it was giving the same CI failure, (mypy check running fine on local though), 
After which i tried typecasting, which in turn gave the same failure.
Didn't change it afterwards

Reply:
What version of mypy are you using locally?

Reply:
The latest available through pip